wandb/cli/cli.py

```diff
@@ -2345,8 +2345,29 @@ def artifact():
     default=None,
     help="Resume the last run from your current directory.",
 )
+@click.option(
+    "--skip_cache",
+    is_flag=True,
+    default=False,
+    help="Skip caching while uploading artifact files.",
+)
+@click.option(
+    "--policy",
+    default="mutable",
+    help="Set the storage policy while uploading artifact files.",
+)
```

Comment:
shouldn't this one be choice option, given that it is a list of acceptable values? https://click.palletsprojects.com/en/8.1.x/options/#choice-options